openapi/openapi.yaml

```diff
@@ -209,6 +209,39 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/Scripts'
+
+  /backplane/script/{clusterId}:
+    get:
+      operationId: getScripts2
+      summary: Get list of backplane scripts the user has access to
```

Comment:
Should the summary here reflect that the call is to the given cluster?

Reply:
@bmeng The semantics of  `/backplane/script/{cluster}` are the same as the `/backplane/script`. 
The `/backplane/script` already takes cluster into consideration, but it is not mentioned in the summary.
To compare with other commands, for example, `/backplane/script/{clusterId}/job`has `clusterId` in path, but the summary:
`get:
      operationId: getAllJobs
      summary: Get all job runs in the namespace the user has access to`
doesn't mention cluster as well.

So I believe the summary should stay the same as the semantics didn't change.

Reply:
`/backplane/script` does NOT take the cluster into consideration as the cluster id is not conveyed on this path.
That said, `/backplane/script/{cluster}` is gonna replace `/backplane/script`, so they could keep the same description indeed... especially if other paths conveying the cluster id do not mention they are specific to a given cluster.